### PR TITLE
[SMP-2216] SMP 0.10.0 809xx release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
SMP 0.10.0 release notes

For reviewers, a preview is available.

[Harness Self-Managed Enterprise Edition Release Notes (NextGen)](https://6542d27fb5e696436fdafce7--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition)

[Harness Self-Managed Enterprise Edition Release Notes (FirstGen)](https://65416136170517008a3164c8--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 1]
```diff
@@ -16,6 +16,61 @@ For Harness SaaS release notes, go to [Harness SaaS Release Notes](/docs/first-g
...
 Release notes are displayed with the most recent release first.
 
+
+## October 31, 2023, patch release for version 809xx
+
+This release includes the following module and component versions.
+
+| **Name** | **Version** |
+| :-- | :-- |
+| Manager | xxxxx |
```
Comment (akshay-khole) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:26]: versions are missing , is this intentional ?

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 2]
```diff
@@ -16,6 +16,61 @@ For Harness SaaS release notes, go to [Harness SaaS Release Notes](/docs/first-g
...
+| Watcher | xxxxx |
+| Verification Service | xxxxx |
+| UI | xxxxx |
+| Learning Engine | xxxxx | 
+| Gateway | xxxxx |
```
Comment (akshay-khole) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:31]: Nit: - this copy paste keeps happening @swapnil-harness  can we standardize in 1 place and then @brian-f-harness  can figure out a way to get it automatically in release notes

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 3]
```diff
@@ -16,6 +16,61 @@ For Harness SaaS release notes, go to [Harness SaaS Release Notes](/docs/first-g
...
+
+- Earlier, you could upload Excel and .csv files in the **File Configuration** dialog. (PL-40796, ZD-44419)
+
+   This issue is fixed. Excel and .csv files in the **File Configuration** dialog are now blocked.
+
+- When you attempted to view the change log of an encrypted text secret, if the resultant API request returned an HTTP 400 response code, the Harness FirstGen user interface failed to show an appropriate message. Instead, it showed the following message and prompted you to refresh your browser page to continue: “Something went wrong... This error has been reported and we are looking into it with high priority.” (PL-40957, ZD-49757)
+
+  This issue has been fixed. The Harness UI now shows you a more appropriate message in the Change Log dialog.
+
+- The **Secrets** dropdown list on the Services page didn't include all available secrets. The list now includes all secrets, up to the maximum of 1000. (PL-41308, ZD-50687)
```
Comment (akshay-khole) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:54]: would customer have access to PL-41308 ?

File: release-notes/self-managed-enterprise-edition.md [hunk 4]
```diff
@@ -13,16 +13,830 @@ import TabItem from '@theme/TabItem';
...
 ### New features and enhancements
 
 This release includes the following Harness module and component versions.
 
+| **Name** | **Version** |
+| :-- | :--: |
+| Helm Chart | [0.10.0](https://github.com/harness/helm-charts/releases/tag/harness-0.10.0) |
+| Air Gap Bundle | [0.10.0](https://console.cloud.google.com/storage/browser/smp-airgap-bundles/harness-0.10.0) |
+| NG Manager | 809xx |
+| CI Manager | xxxx |
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md:32]: versions missing

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:26]: @akshay-khole Yes, I normally don't have access to the versions until the Helm chart is published. If there's a way to view them earlier for draft release notes, I'll add them.

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md (hunk 3 above)
Comment (brian-f-harness) [reply docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:54]: @akshay-khole Access to the PL jira? We use the Jira numbers for internal reference for all release notes. We also include any applicable zen desk tickets.

File: release-notes/self-managed-enterprise-edition.md (hunk 4 above)
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md:32]: @swapnil-harness If these versions are available, please let me know. I normally add the versions when the Helm chart is published.

**Review by swapnil-harness — DISMISSED** [review]

**Pull request merged**